### PR TITLE
Update vpn-device-tunnel-config.md

### PR DESCRIPTION
**Pull request by msarcletti**
Adding the support statement for NRPT usage with device tunnel as discussed with PG.

### CODE REVIEW

**Comment by kaushika-msft** [comment]
# sign-off

**Comment by Teresa-Motiv** [comment]
@kaushika-msft I don't have access to edit this without creating a child PR, but a couple of tweaks to the last sentence would help:
Device tunnel does not support Force tunnel. You must configure it as Split tunnel.

File: WindowsServerDocs/remote/remote-access/vpn/vpn-device-tunnel-config.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Unlike _user tunnel_, which only connects after a user logs on to the device or
 
 User tunnel is supported on domain-joined, nondomain-joined (workgroup), or Azure AD–joined devices to allow for both enterprise and BYOD scenarios. It is available in all Windows editions, and the platform features are available to third parties by way of UWP VPN plug-in support.
 
-Device tunnel can only be configured on domain-joined devices running Windows 10 Enterprise or Education version 1709 or later. There is no support for third-party control of the device tunnel.
+Device tunnel can only be configured on domain-joined devices running Windows 10 Enterprise or Education version 1709 or later. There is no support for third-party control of the device tunnel. Device tunnel does not support using the Name Resolution Policy table (NRPT). Device Tunnel does not support Force Tunnel, it must be configured as Split Tunnel.
```
Comment (illfated) [inline WindowsServerDocs/remote/remote-access/vpn/vpn-device-tunnel-config.md:27]:
```suggestion
Device tunnel can only be configured on domain-joined devices running Windows 10 Enterprise or Education version 1709 or later. There is no support for third-party control of the device tunnel. Device tunnel does not support using the Name Resolution Policy table (NRPT). Device tunnel does not support Force tunnel. You must configure it as Split tunnel.
```

Comment (illfated) [reply WindowsServerDocs/remote/remote-access/vpn/vpn-device-tunnel-config.md:27]: (just to visualize the recommendation from Teresa-Motiv)

**Comment by illfated** [comment]
#sign-off

**Comment by Teresa-Motiv** [comment]
@eross-msft , can you merge this PR? It's been stuck for quite a while.

**Comment by Teresa-Motiv** [comment]
@eross-msft Thanks!

**Pull request merged**